Vizzuality/half-earth-v3
# Fix/arrow color

## Turn the arrow in challenges x-axis white
### Description
The arrow was hardly visible due to the color
### Designs
https://www.figma.com/file/2YAp3AS5lYCj2IWEAN8AJw/HE-Map---Work-in-Progress?node-id=4%3A14646
### Testing instructions
Go to the challenges chart on any NRC country. You should be able to see the charts
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-405?atlOrigin=eyJpIjoiYjE0YjBmYTRmNzNhNDVjYzlkMGRjYTNhNmNhNDk0M2YiLCJwIjoiaiJ9
